content/en/docs/tasks/administer-cluster/dns-custom-nameservers.md

```diff
@@ -48,10 +48,10 @@ lookups (PTR records).
 
 ## Inheriting DNS from the node
 
-When running a pod, kubelet will prepend the cluster DNS server and search
-paths to the node's own DNS settings.  If the node is able to resolve DNS names
-specific to the larger environment, pods should be able to, also.
-See [Known issues](#known-issues) below for a caveat.
+If a pod's `dnsPolicy` is set to "`default`", it will inherit the name
+resolution configuration from the node that the pods run on. If the node is able
+to resolve DNS names specific to the larger environment, pods should be able to,
+also. See [Known issues](/docs/tasks/administer-cluster/dns-debugging-resolution/#known-issues) for a caveat.
```

Comment:
why change to this url? i think origin link that's ok

Reply:
There's no anchor named "Known issues" in [dns-custom-nameservers](https://kubernetes.io/docs/tasks/administer-cluster/dns-custom-nameservers/), it was moved to [dns-debugging-resolution](https://kubernetes.io/docs/tasks/administer-cluster/dns-debugging-resolution/) at some time.